galaxy.yml

```diff
@@ -34,7 +35,14 @@ license_file: LICENSE
 
 # A list of tags you want to associate with the collection for indexing/searching. A tag name has the same character
 # requirements as 'namespace' and 'name'
-tags: []
+tags:
+  - crowdstrike
+  - falcon
+  - sensor
+  - linux
+  - windows
+  - macOS
+  - security
```

Comment:
```suggestion
  - crowdstrike
  - falcon
  - sensor
  - linux
  - windows
  - macOS
  - security
  - edr
  - xdr
```

Reply:
Should we add EDR? Yahhh it's a little markety but why not?

Reply:
Sure... added